ntbs-service/Pages/Notifications/LinkedNotifications.cshtml

```diff
@@ -7,6 +7,7 @@
     ViewData["Title"] = "Linked Notifications";
     ViewData["NotificationId"] = @Model.NotificationId;
     ViewData["CurrentPage"] = NotificationSubPaths.LinkedNotifications;
+    TempData["OnLinkedNotificationsPage"] = true;

```

Comment:
So what about this solution?
I am using TempData to see if I am on LinkedNotifications Page so that I can display edit details link only on that page even if it's Draft